source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -550,6 +551,41 @@
   "test.environment.shelleyTestnetLabel": "Shelleyテストネット",
   "test.environment.stagingLabel": "ステージング",
   "test.environment.testnetLabel": "テストネット",
+  "voting.info.buttonLabel": "Register to vote",
+  "voting.info.description": "!!!To start yo will need to have the <span>Catalyst Voting App</span> ready on your smart phone! To register to vote you will need a <span>PIN code</span> and a <span>QR code</span> that you will generate here within <span>Daedalus</span>. Before you start",
```

Comment:
@juan-cieri I can see all Japanese translations are missing?

Reply:
Were can we find these Japanese translations?

Reply:
@juan-cieri please create a document such as this one: https://docs.google.com/document/d/1GNPauihmJDvXtMHrs0gTaUe8gPwniwlfdUDnpHuKaXw/edit# and post it on Slack.